android-json-form-wizard/src/main/java/com/vijay/jsonwizard/customviews/NumericDatePicker.java

```diff
@@ -86,7 +87,7 @@ protected void resetDatePicker() {
             switch (violation) {
                 case MALFORMED_DATE:
                     // Only Day can malform since different months have different days in the Gregorian calendar
```

Comment:
We can remove this comment, the month could potentially lead to this state as well